packages/common/src/cis2/util.ts

```diff
@@ -30,6 +35,19 @@ export namespace CIS2 {
      */
     export type Address = ContractAddress | Base58String;
 
+    /**
+     * A Token ID
+     */
+    export type TokenId = HexString;
+
+    /**
+     * A Token Address
```

Comment:
Expand a bit on what this is. Rephrasing the type name in a comment isn't very useful.